style.css

```diff
@@ -0,0 +1,812 @@
+  width: 85vw;
+}
+
+.productsList {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100vw;
+  margin: 0;
+  position: relative;
+  justify-content: center;
+}
+
+.products {
+  border: #666 2px solid;
+  border-radius: 10px;
+  font-size: 40px;
+  margin: 0;
+  padding: 0;
+  margin-bottom: 20px;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.prd1,
+.pdr2 {
+  display: flex;
+  width: 95vw;
+  border: 1px rgba(255, 0, 0, 0) solid;
+  background-color: #bfbfbf75;
+  position: relative;
+  flex-direction: row;
+  align-items: center;
+  align-content: flex-start;
+  justify-content: space-evenly;
+}
+
+.svgImage {
+  margin: 10px;
+  width: 8vw;
+  height: 7vh;
+}
+
+.productImage {
+  margin: 20px;
+  width: 35vw;
+}
+
+.productImageBoard {
+  margin: 5px;
+  position: absolute;
+  width: 20vw;
+  z-index: -1;
+}
+
+.prodText {
+  text-shadow: 0 0 4px #000;
+  color: #fff;
+  width: 50vw;
+}
+
+.realProduct {
+  display: flex;
+  flex-direction: row;
+}
+
+.realProdSection {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.arrow {
+  border: solid #ec5242;
+  border-width: 0 3px 3px 0;
+  display: inline-block;
+  padding: 3px;
+}
+
+.down {
+  transform: rotate(45deg);
+  -webkit-transform: rotate(45deg);
+}
+
+.up {
+  transform: rotate(-135deg);
+  -webkit-transform: rotate(-135deg);
+}
+
+.productDescription {
+  display: flex;
+  flex-direction: column;
+}
+
+#ourProductsH2 {
+  margin: 0;
+  margin-top: 30px;
+  text-align: center;
+}
+
+hr {
+  width: 20vw;
+  background-color: #ec5242;
+  height: 1.5px;
+  margin-bottom: 5vh;
+}
+
+.moreBtn {
+  border: 2px solid #dbdbdb;
+  font-size: 25px;
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: center;
+  background-color: white;
+  width: 90vw;
+  height: 54px;
+  align-content: center;
+  align-items: center;
+  margin-block: 20px;
+}
+
+.moreBtn:active {
+  color: #dbdbdb;
+  background-color: #ec5242;
+}
+
+.productsBtn {
+  font-family: 'Lato', sans-serif;
+  border-radius: 5px;
+  font-size: large;
+  width: 70vw;
+  height: 50px;
+  background-color: #ec5242;
+  color: #fff;
+  border: none;
+  transition: all 0.4s ease-in-out;
+  margin: 10px;
+}
+
+.moreProd {
+  display: none;
+  flex-direction: column;
+}
+
+.moreShow {
+  display: flex;
+}
+
+.logoAbout {
+  border-top: 1px solid #d3d3d3;
+  border-bottom: 1px solid #d3d3d3;
+}
+
+.eventPhotos {
+  width: 100vw;
+  position: relative;
+}
+
+.secSectionAbout {
+  color: #272a31;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.textImage1 {
+  color: #ec5242;
+  position: absolute;
+  z-index: 1;
+}
+
+.allProductsDesktop {
+  display: none;
+}
+
+svg {
+  color: #ec5242;
+}
+
+.partnerLogos {
+  flex-direction: row;
+  display: flex;
+  width: 100vw;
+  flex-wrap: wrap;
+  align-content: center;
+  justify-content: center;
+  align-items: center;
+  background-color: #d3d3d3;
+}
```

Comment:
- Just as the previous reviewer mentioned, please stick to the design to meet the project's requirements, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
- Please fix this for both the home and about pages, and both mobile and desktop versions 🙏 
<img width="960" alt="rykykryk" src="https://user-images.githubusercontent.com/73607512/210561100-3d7726f9-b000-41c8-9b3b-acdb691f82d6.PNG">
Figma Design:
![210536830-37491abd-8b5b-40da-ba41-b0f1c1686287](https://user-images.githubusercontent.com/73607512/210561239-a4274999-500b-49a9-89b4-50fc76193c32.png)

Reply:
Hi Please explain like what to change the brand icons or some styling design such as background color>

Reply:
hey @anas599 , please change the background color so it matches the color in the design template, sorry for my late response 🙏